x/wasm-storage/keeper/msg_server.go

```diff
@@ -25,6 +26,11 @@ func NewMsgServerImpl(keeper Keeper) types.MsgServer {
 
 var _ types.MsgServer = msgServer{}
 
+var OracleProgramCountMetric = prometheus.NewCounter(prometheus.CounterOpts{
+	Name: "seda_oracle_program_count",
+	Help: "The count of stored oracle programs",
+})
+
```

Comment:
Wouldn't it be nice to also track the total size of oracle programs stored in our chain?